From 973603afc707f1f470ac343fb96edac91cc16ca6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <bostik@odin.bostik.iki.fi>
Date: Fri, 12 Oct 2012 08:25:15 +0300
Subject: [PATCH 2/9] Remove tests from build

Disable tests from build. We assume upstream has run them, and for us
they do nothing except prolong an already lengthy build.
---
 Source/QtWebKit.pro |    6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/Source/QtWebKit.pro b/Source/QtWebKit.pro
index 19d52fc..a1da51e 100644
--- a/Source/QtWebKit.pro
+++ b/Source/QtWebKit.pro
@@ -33,9 +33,5 @@ build?(webkit1) {
         tests.file = tests.pri
         SUBDIRS += tests
     }
-
-    examples.file = WebKit/qt/examples/examples.pro
-    examples.CONFIG += no_default_target
-    examples.makefile = Makefile
-    SUBDIRS += examples
 }
+
-- 
1.7.10.4

